### ISSUE TITLE
building:part=yes not rendered if only roof is visible

### ISSUE THREAD

**Issue by Nadyita** [tordanik/OSM2World#203]
Common scenario: I have a building-passage which is covered by a roof only. When I set the following tags:
```
building:part=yes
building:levels=1
building:min_level=1
roof:levels=1
roof:shape=gabled
roof:orientation=along
roof:height=2.2
height=5
```
it doesn't get rendered at all. Same when using `min_height=2.8`. It works when I set `building:part=roof` instead, but that will give me an "unfilled" roof, not the roof of a house.
There are many use-cases when I need something like this, most are building-passages, or just the roof not covering all of the house and the "body" of the house consists of my elements without a roof and then finally the roof is added.

**Comment by tordanik** [comment]
Thanks for reporting this, I believe I've noticed some similar problems and this might help us get to the bottom of it.

However, I have some difficulty reproducing the issue:

![issue-203](https://user-images.githubusercontent.com/1270237/200353047-f51c63fd-1f21-4195-9bae-9b11c2617271.png)

The central building part in this example has the tags you provided, and it does get rendered. Can you modify the [example file](https://github.com/tordanik/OSM2World/files/9953224/issue-203.zip) or provide a different one to trigger the error?

**Comment by Nadyita** [comment]
Here's ax [example.zip](https://github.com/tordanik/OSM2World/files/9959611/example.zip) where it doesn't work. I think I already figured out, how to trigger it.

![image](https://user-images.githubusercontent.com/43058421/200526927-8127bc1d-a6e6-4fcc-9bfc-39c1d287ada2.png)

This node is shared by all 3 building parts. If you remove it from the middle roof, it gets rendered again and all is well.

**Comment by tordanik** [comment]
This is now fixed in 9a94a98. Thank you for the excellent test case! :+1: 

![Rendering of the latest test case](https://user-images.githubusercontent.com/1270237/209644957-10281942-b611-4f79-b38b-5cef2c507829.png)
